apps.toml

```diff
@@ -2465,6 +2465,11 @@ state = "working"
 subtags = [ "website" ]
 url = "https://github.com/YunoHost-Apps/pelican_ynh"
 
+[penpot]
+category = "productivity_and_management"
```

Comment:
i think `dev` would be more suitable

Reply:
`productivity_and_management` seems fine to me.

Reply:
But it’s literally a dev tool, to design apps or websites x)

Reply:
Well i kinda disagree, i'd say it's closer to a drawing too than to a development tool ^^''